parser/MyParser.java

```diff
@@ -63,6 +62,7 @@
 
   // Stanford NLP parser variables
   private final static String PCG_MODEL = "edu/stanford/nlp/models/lexparser/englishPCFG.ser.gz";
+  // private final static String PCG_MODEL = "deps/englishPCFG.ser.gz";
```

Comment:
This is where I kept my PCG_MODEL relative to the code.  I'll delete this line for cleanliness, though it would be great to add some option to the arguments to specify a PCFG outside of an `deu/stanford/nlp/models/lexparser` directory.